tests_tf/test_attacks.py

```diff
@@ -1255,3 +1227,7 @@ def test_keep_pixels_on_image(self):
     new_labs = np.argmax(self.sess.run(self.model.get_logits(x_adv)), axis=1)
     print(np.mean(old_labs == new_labs))
     self.assertTrue(np.mean(old_labs == new_labs) < 0.3)
+
+if __name__ == "__main__":
```

Comment:
I thought we were removing these statements because they prevent nosetest from correctly supporting "nosetests foo.py:bar" syntax to run only `bar`

Reply:
Done